solution/serverless-test-files/serverless-backend.yml

```diff
@@ -0,0 +1,237 @@
+service: cmcs-eregs-site
+variablesResolutionMode: 20210326
+
+provider:
+  name: aws
+  runtime: python3.8
+  region: us-east-1
+  iam:
+    role: LambdaFunctionRole
+  lambdaHashingVersion: '20201221'
+  environment:
+    DB_NAME: eregstest 
+    DB_USER: eregstest1 
+    DB_PASSWORD: ${ssm:/eregulations/db/password}
+    DB_HOST: !GetAtt [RDSResource, Endpoint.Address]
+    DB_PORT: !GetAtt [RDSResource, Endpoint.Port]
+    #SURVEY_URL: ${ssm:/eregulations/survey_url}
+  vpc:
+    securityGroupIds:
+      - !Ref ServerlessSecurityGroup
+    subnetIds:
+      - subnet-021d6d373ec2d8c86
+      - subnet-0121c25b7306fe803
```

Comment:
I'm getting this error
Serverless: Operation failed!
Serverless: View the full error output: https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stack/detail?stackId=arn%3Aaws%3Acloudformation%3Aus-east-1%3A666435484074%3Astack%2Fcmcs-eregs-site-dev%2F05d423d0-a665-11ed-934b-0e74c018307b

 Serverless Error ----------------------------------------

  An error occurred: DBSubnetGroup - Resource handler returned message: "Some input subnets in :[subnet-021d6d373ec2d8c86, subnet-0121c25b7306fe803] are invalid. (Service: Rds, Status Code: 400, Request ID: 4245a37d-7cc9-4ef7-9ea8-7e8e51053bae)" (RequestToken: d0d2e84b-c918-109f-5991-e1957db5c04f, HandlerErrorCode: InvalidRequest).

Reply:
changed it to create the subnets as opposed to just trying to call existing one.